packages/core/src/test/shared/sam/build.test.ts

```diff
@@ -229,6 +232,46 @@ describe('SAM build helper functions', () => {
             assert.deepStrictEqual(quickPick.items, expectedItems)
         })
     })
+
+    describe('resolveBuildFlags', () => {
+        let sandbox: sinon.SinonSandbox
+        let pathAndVersionStub: sinon.SinonStub<any[], any>
+
+        beforeEach(() => {
+            sandbox = sinon.createSandbox()
+        })
+
+        afterEach(() => {
+            sandbox.restore()
+        })
+
+        it('should add --no-use-container if the buildFlags array does not contain --use-container', async () => {
```

Comment:
this title seems really into the weeds with the implementation. Maybe something like `uses --no-use-container when --use-container is absent`